src/chocolatey.tests.integration/scenarios/InstallScenarios.cs

```diff
@@ -3390,6 +3598,154 @@ public void should_not_install_any_packages()
             {
                 Results.Count().ShouldEqual(0);
             }
+
+            [Concern(typeof(ChocolateyInstallCommand))]
+            public class when_installing_new_package_from_priority_source_with_repository_optimization : ScenariosBase

```

Comment:
This is a rebase error, need to take out the duplicated tests from this before going any further.